test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetNewTemplatesTests.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Xml.Linq;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace Dotnet.Integration.Test
+{
+    [Collection("Dotnet Integration Tests")]
+    public class DotnetNewTemplatesTests
+    {
+        private readonly MsbuildIntegrationTestFixture _fixture;
+
+        public DotnetNewTemplatesTests(MsbuildIntegrationTestFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [PlatformFact(Platform.Linux)]
+        public void Dotnet_New_ConsoleApp_Pack_Success()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var projectName = new DirectoryInfo(pathContext.SolutionRoot).Name;
+                var projectDirectory = pathContext.SolutionRoot;
+                var nupkgPath = Path.Combine(projectDirectory, "bin", "Debug", $"{projectName}.1.0.0.nupkg");
+
+                // Act
+                CommandRunnerResult newResult = _fixture.RunDotnet(pathContext.SolutionRoot, "new console");
```

Comment:
nit: This would throw here if it fails, thus making the assertions at line 38 and line 41 moot.